### PR TITLE
Import Linux Kernel typos: letter T

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -11316,6 +11318,7 @@ tanget->tangent
 tangetial->tangential
 tangetially->tangentially
 tangets->tangents
+tanks->thanks
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: tanks is a valid word

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -11490,13 +11508,15 @@ thimngs->things
...
 thise->these
+thist->this
 thiunk->think
 thjese->these
 thn->then
 thna->than
 thne->then
 thnig->thing
 thnigs->things
+thorugh->through
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: can also be `thorough`

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Removed

File: codespell_lib/data/dictionary.txt (hunk 2 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Added.

**Pull request merged**